### DISCUSSION TITLE
Display additional Attributes without having to Edit

### DISCUSSION THREAD

**Discussion by psyciknz** [keepassxreboot/keepassxc#11443] [Ideas]
I make use of additional attributes for things like credit cards, where the additional attributes might contain the expiry year/moths etc.

Sometimes if I want that information, using the copy attribute feature is not possible due to the way they have structured their forms.  Ex Expiry Month and the web page has a drop down list.

So in this scenario I generally have to go into edit, then attributes and view each one.   This makes me nervous that I'll update or change something.

So what I'm after is a simple dialog where it will show all these parameters, but in a read only settings.  Somethign akin to the Show TOTP operation would be perfect.  Where I can just read all these attibutes.

**Comment by droidmonkey** [comment]
The preview panel, advanced tab, is exactly what you are looking for https://keepassxc.org/docs/KeePassXC_UserGuide#_application_layout

**Comment by psyciknz** [comment]
But I'm editing the entry at that point.  Having that viewable off the right context menu without going to edit first would be the preference.

**Reply by droidmonkey** [reply]
The preview panel is not visible when editing an entry. Please read the documentation i linked.

**Comment by psyciknz** [comment]
ahh right, yes I had the preview pane split all the way to the bottom, so it was not visible.

Yes, consider this closed.
